test/acceptance/renumber.js

```diff
@@ -0,0 +1,12 @@
+describe('todo renumber', function() {
```

Comment:
@vesln what's the general difference between these acceptance tests and the test file like `/test/todos.js`?